docs/Helix.md

```diff
@@ -36,8 +51,6 @@ There's a link embedded in the build.cmd log of the helix target on Azure Pipeli
 
 The link will take you to an overview of all the tests with clickable links to the logs and each run broken down by queue.
 
-All of the helix runs for aspnetcore can be found here <https://mc.dot.net/#/user/aspnetcore/builds>.
-
 ## What do I do if a test fails?
 
 You can simulate how most tests run locally:
```

Comment:
- Do the instructions below still match how we run Helix jobs❔
- Should something in here mention eng/scripts/RunHelix.ps1 and does that script work correctly these days (I had trouble using it a week or so ago)❔

Reply:
>     * does that script work correctly these days

I used it yesterday and it worked. I think there might be an issue when you try to select queues.

Reply:
Well, fixing such an issue sounds worthwhile but shouldn't prevent us from documenting it as @HaoK has done 😺